service/dap/server.go

```diff
@@ -1429,7 +1444,7 @@ func (s *Server) sendErrorResponseWithOpts(request dap.Request, id int, summary,
 	er.Body.Error.Id = id
 	er.Body.Error.Format = fmt.Sprintf("%s: %s", summary, details)
 	er.Body.Error.ShowUser = showUser
-	s.log.Error(er.Body.Error.Format)
+	s.log.Debug(er.Body.Error.Format)
```

Comment:
Why hide the error behind the logging flags?
Don't the users turn to the debug console when something doesn't work as expected because things are not always clearly reflected in the UI?

Reply:
Users are supposed to know the issue from the error response, not from the server-side logging.